### PR TITLE
Add workshop content to docs site

### PR DESCRIPTION
**Pull request by bsatrom**
@interested1 and @rickkas7 this is a *draft* PR to add a Workshops section to the docs site. It's not done and needs the following before we can merge:

- [x] Configure Left Nav for Workshops Section
- [x] Move images into `assets` and update image links in content
- [x] Configure header tables on section pages
- [x] review and tweak main workshop landing page
- [x] fix the keynote slide embeds

adding this now for our discussion and your review. Feel free to add any input here.

### CODE REVIEW

**Comment by bsatrom** [comment]
Additional actions based on feedback from @interested1 

- [x] Add some text to each slide embed page so it doesn't look like an empty page
- [x] Make sure the step numbering is correct
- [x] Tweak the TOC with shorter names for the sidebar
- [x] Edit the Workshops HBS to skip creating a main nav for the first folder
- [x] Add more additional content to the main workshops landing page

**Comment by bsatrom** [comment]
@interested1 @rickkas7 marking this one as ready to review and potentially merge. LMK if you have any questions!

File: scripts/metalsmith.js [hunk 1]
```diff
@@ -184,6 +184,14 @@ exports.metalsmith = function() {
           'hardware-projects'
         ]
       },
+      workshops: {
```
Comment (rickkas7) [inline scripts/metalsmith.js]: There are two workshops sections. One here and one on line 205. I don't think you want both.

**Review by rickkas7 — APPROVED** [review]
Double check that one thing on metalsmith.js, otherwise looks good. I reviewed all of the technical changes and they look fine. I ran the modified version and the navigation looks good but I didn't look through all of the content.

**Review by interested1 — CHANGES_REQUESTED** [review]
OK, I believe I've pulled the most recent version and am on the correct branch. Here are a few things I found that need to be fixed before going live. 

- First off, the navigation looks great, Brandon. 

- Loading the slides still takes a bit of time — not sure if it's my internet connection or simply a slow load. http://localhost:8080/workshops/mesh-101-workshop/a-lap-around-particle/ was one of the slow loaders for me.

- Lack of step enumeration in the following sections: 
http://localhost:8080/workshops/photon-maker-kit-workshop/ch1/#mobile-app-instructions
AND http://localhost:8080/workshops/photon-maker-kit-workshop/ch2/
AND http://localhost:8080/workshops/photon-maker-kit-workshop/ch3/
AND http://localhost:8080/workshops/photon-maker-kit-workshop/ch4/
AND http://localhost:8080/workshops/photon-maker-kit-workshop/ch5/.

- Lack of step enumeration and code improperly formatted: http://localhost:8080/workshops/mesh-101-workshop/mesh-button-led/#remote-display

**Comment by bsatrom** [comment]
@interested1 not really much I can do about the slow-loading slides. That's all on Apple...

I'll take a look at those other items now

**Pull request merged**